llvm/test/CodeGen/RISCV/attributes-module-flag.ll

```diff
@@ -0,0 +1,17 @@
+; RUN: llc -mtriple=riscv32 %s -o - | FileCheck %s --check-prefix=RV32
+; RUN: llc -mtriple=riscv64 %s -o - | FileCheck %s --check-prefix=RV64
+
+; Test generation of ELF attribute from module metadata
+
+; RV32: .attribute 5, "rv32i2p1_m2p0_zba1p0"
+; RV64: .attribute 5, "rv64i2p1_m2p0_zba1p0"
```

Comment:
I want to double check that we don't need to test the merging w/  `llvm-lto2`, do we? I //think// this test is equivalent to what would happen in that case, so I believe we're OK, but I'm not 100%.

Reply:
My intent was that this test is what would happen in that case. I can add an llvm-lto2 test for the merging if reviewers want.

Reply:
I don’t think that’s necessary. I’m pretty sure you’re right and this is equivalent.